### PR TITLE
Adding transferbench to the list of components

### PR DESCRIPTION
**Pull request by SwRaw**

### CODE REVIEW

**Review by randyh62 — DISMISSED** [review]

**Review by lpaoletti — COMMENTED** [review]
Both of these additions have not been ordered alphabetically and must be fixed. 

The change to the What is ROCm stacked image will be tracked in a different Jira ticket.

File: docs/what-is-rocm.rst [hunk 1]
```diff
@@ -116,6 +116,7 @@ Performance
   ":doc:`Omniperf <omniperf:index>`", "System performance profiling tool for machine learning and HPC workloads"
   ":doc:`Omnitrace <omnitrace:index>`", "Comprehensive profiling and tracing tool for HIP applications"
   ":doc:`ROCm Bandwidth Test <rocm_bandwidth_test:index>`", "Captures the performance characteristics of buffer copying and kernel read/write operations"
+  ":doc:`TransferBench <transferbench:index>`", "Benchmarks simultaneous transfers between user-specified devices"
```
Comment (lpaoletti) [inline docs/what-is-rocm.rst:119]: This description should match the first line in the docs - or be a closer in terms of details. 
TransferBench is a utility to benchmark simultaneous transfers between user-specified devices (CPUs or GPUs).

Comment (SwRaw) [reply docs/what-is-rocm.rst]: @lpaoletti I have derived the description from the first line in the doc. I noticed the description of other components in the list start with the utility of the tool hence I kept it in the same format.

Let me know if I should change it to "Utility to benchmark simultaneous transfers between user-specified devices (CPUs or GPUs)".

**Comment by SwRaw** [comment]
> Both of these additions have not been ordered alphabetically and must be fixed.
> 
> The change to the What is ROCm stacked image will be tracked in a different Jira ticket.

Done!

**Review by peterjunpark — DISMISSED** [review]
LGTM

**Comment by SwRaw** [comment]
This needs to be released in 6.3 hence moving the PR to rocm-internal.

**Pull request closed without merging**